app/services/bulk_tagging/document_type_tagger.rb

```diff
@@ -1,22 +1,20 @@
 module BulkTagging
   class DocumentTypeTagger
-    def self.call(taxon_base_path:, document_type:)
-      new(taxon_base_path: taxon_base_path, document_type: document_type).call
+    def self.call(taxon_content_id:, document_type:)
+      new(taxon_content_id: taxon_content_id, document_type: document_type).call
     end
 
-    def initialize(taxon_base_path:, document_type:)
-      @taxon_base_path = taxon_base_path
+    def initialize(taxon_content_id:, document_type:)
+      @taxon_content_id = taxon_content_id
       @document_type = document_type
     end
 
     def call
-      taxon_content_id = Services.publishing_api.lookup_content_id(base_path: @taxon_base_path)
-      raise StandardError, "Cannot find taxon with base path #{@taxon_base_path}" if taxon_content_id.nil?
-
+      Services.publishing_api.get_content(@taxon_content_id)
```

Comment:
This call now looks pretty redundant, though I guess it will error if the provided taxon doesn't exist?

Reply:
thats why it is there

Reply:
I'd rather we explicitly raise the error here if the taxon doesn't exist, the previous implementation documented what error case we were covering. However in this addition, it's not obvious as to why we're asking the publishing API for some content but not using its response.

I think this could be equivalent but shows that we're expecting a `HTTPNotFound` error to potentially be raised.

```ruby
begin
  Services.publishing_api.get_content(@taxon_content_id)
rescue GdsApi::HTTPNotFound => e
  raise
end

Reply:
You can actually link to things which don't exist, but this is probably a good feature. A comment saying why this is there might be helpful though?